engine/crates/wasi-component-loader/src/hooks/authorization.rs

```diff
@@ -210,7 +222,10 @@ impl AuthorizationComponentInstance {
         edges: Vec<(String, Vec<String>)>,
         metadata: String,
     ) -> crate::Result<Vec<Result<(), crate::GuestError>>> {
+        let span = tracing::info_span!(AUTHORIZE_EDGE_POST_EXECUTION_HOOK_FUNCTION);
+
         self.call3_one_output(
```

Comment:
maybe we should add the instrument to the `call3_one` etc. variants? we would just name to set `otel.name` to the function name so that hey appear correctly on otel data

Reply:
is otel.name then the value that gets shown in our tracing UI?

Reply:
yeah it's the span name that will be exported by gateway

Reply:
it overrides the tracing span name if present

Reply:
Awesome, then there's no const restriction and I can send a dynamic string...